doc/reference/power_management/index.rst

```diff
@@ -492,10 +492,6 @@ the following configuration flags.
    This flag is enabled if the SOC interface and the devices support device power
    management.
 
-:code:`CONFIG_PM_DEVICE_IDLE`
```

Comment:
Comparing to Linux, this flag enables what they call "runtime PM". It may still be useful to have a distinction between device PM (used by system PM) and runtime device PM. In some cases we may want to have one but not the other, e.g. I want that all devices are suspended when I suspend the sytem, but I don't care about PM when on running (all `device_pm_get/put` should be a nop). We can discuss this again tomorrow.